docs/lib/sourcing.ts

```diff
@@ -30,6 +30,10 @@ export const getBlogPost = (filePath: string): BlogPost => {
   };
 };
 
+// Avoid typos in the blog markdown pages.
+// https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917
+const ALLOWED_TAGS = ['MUI Core', 'News', 'MUI X', 'Company', 'All products'];
```

Comment:
Feels like this:

<img width="685" alt="Screenshot 2022-02-05 at 15 14 59" src="https://user-images.githubusercontent.com/3165635/152645650-46bc0797-aef8-4cd8-baf7-e6dbbbf08eea.png">